content/blog/2023/2023-08-23-fairness-dataset-bias.md

```diff
@@ -0,0 +1,74 @@
+
+Recognizing these challenges, there has been a surge in efforts to mitigate this bias. The outcome has been the development of models that aim for balanced outcomes for all demographic groups or maintain consistent true positive rates across them. Mitigating bias might mean a slight dip in overall accuracy, but it is a necessary trade-off to ensure fairness in our models. Using fairness algorithms is ethical and highly beneficial to ensure people's trust in machine learning and AI.
+
+## Introducing the Fairness add-on in Orange
+
+Hopefully, the above has emphasized the importance of fairness in machine learning. With this in mind, let us discuss the new Orange fairness add-on. It aims to empower users with tools to detect and mitigate bias in machine-learning workflows. At the time of writing, the add-on includes seven widgets, four of which are used for bias detection and mitigation. The remaining three are used to support the other widgets and provide additional functionality.
+
+In this blog and the following ones, we will familiarize ourselves with the add-on widgets and their use cases. We will also demonstrate how to use them in real-world scenarios and explain relevant concepts. Let us take a look at the first two widgets in more detail.
+
+## Exploring the Widgets: Dataset Bias and As Fairness
+
+This blog will introduce the first two fairness widgets: As Fairness Data and Dataset Bias. Both aim to address the different bias identification and correction stages in machine learning workflows.
+
+
+#### 1. As Fairness Widget:
+
+The As Fairness Widget does not do anything about fairness on its own. Instead, it allows users to designate fairness attributes in the dataset, which are essential for other fairness algorithms to operate.
+
+When inputting a dataset into the widget, one will see options for three attributes:
+
+- Favorable Class Value: Define which class value is viewed as favorable.
+
+- Protected Attribute: Select the dataset feature representing or containing potentially biased groups or those for which fair predictions are desired, such as race, gender, or age.
+
+- Privileged Protected Attribute Values: Specify values within the protected attribute deemed privileged.
+
+{{< window-screenshot src="/blog_img/2023/2023-08-23-fairness-as-fairness-data.png" >}}
+
+#### 2. Dataset Bias Widget:
+
+Before training a model, it is crucial to understand if the data itself is skewed. The Dataset Bias widget aids in precisely that. After giving it a dataset as input, the widget will calculate two fairness metrics according to the dataset:
+
+- Disparate Impact (DI): Measures the ratio of favorable outcomes for an unprivileged group to that of the privileged group. An ideal value of 1.0 means the ratio of favorable outcomes is the same for both groups.
```

Comment:
Could you also link some reference if anyone wants to read more about these measures?